### PR TITLE
Adding new dashboard JSON and updating the screenshots for grafana

### PR DESCRIPTION
**Pull request by cloudbehl**
#tendrl-bug-id: https://github.com/Tendrl/monitoring-integration/issues/36

### CODE REVIEW

**Comment by cloudbehl** [comment]
I am done with one round of testing.

@rishubhjain can you also do the testing with the new dashboards JSON. 

@r0h4n Once @rishubhjain provides ack then we can go ahead and merge this patch.

File: etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-at-a-glance.json [hunk 1]
```diff
@@ -1,5 +1,4 @@
 {
- "dashboard":{
   "annotations": {
```
Comment (rishubhjain) [inline etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-at-a-glance.json]: @cloudbehl Please add "  "dashboard" : {  " before "annotations"

File: etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-bricks.json [hunk 2]
```diff
@@ -1,34 +1,291 @@
 {
-   "dashboard":{  
- "annotations": {
+  "annotations": {
```
Comment (rishubhjain) [inline etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-bricks.json]: @cloudbehl Please add "  "dashboard" : {  " before "annotations"

File: etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-hosts.json [hunk 3]
```diff
@@ -1,6 +1,5 @@
 {
-   "dashboard":{  
- "annotations": {
+  "annotations": {
```
Comment (rishubhjain) [inline etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-hosts.json]: @cloudbehl Please add "  "dashboard" : {  " before "annotations"

File: etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-volumes.json [hunk 4]
```diff
@@ -1,19 +1,18 @@
 {
-  "dashboard":{  
   "annotations": {
```
Comment (rishubhjain) [inline etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-volumes.json]: @cloudbehl Please add "  "dashboard" : {  " before "annotations"

File: etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-volumes.json [hunk 5]
```diff
@@ -1,428 +1,2689 @@
...
-          "datasource": "Graphite",
-          "decimals": 0,
-          "format": "short",
-          "gauge": {
-            "maxValue": 100,
-            "minValue": 0,
-            "show": false,
-            "thresholdLabels": false,
-            "thresholdMarkers": true
+  "datasource": {
```
Comment (rishubhjain) [inline etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-volumes.json]: @cloudbehl  datasource - > dashboard

**Comment by rishubhjain** [comment]
@cloudbehl @r0h4n LGTM

**Pull request merged**